scaffolding-chef-infra/lib/linux/scaffolding.sh

```diff
@@ -6,7 +6,7 @@ fi
 scaffolding_load() {
   : "${scaffold_chef_client:=chef/chef-client}"
   : "${scaffold_chef_dk:=chef/chef-dk}"
-  : "${scaffold_cacerts:=}"
+  : "${scaffold_cacerts:=core/cacerts}"
```

Comment:
Setting the default here on `scaffolding_load` always overrides user input. We should set the default later during the run, perhaps in `do_build()` and check to see if it's null at that time.

Reply:
We talked about this in the standup but putting a note here for history. This works on Linux it's on Windows that this was an issue. but because we are setting this to `core/cacerts` here I need to fix the `pkg_deps` loading below this.